mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java

```diff
@@ -57,6 +58,10 @@
     PopularBankAdapter mPopularBankAdapter;
     @Inject
     OtherBankAdapter mOtherBankAdapter;
+    @OnClick(R.id.et_cross_bank)
```

Comment:
functions are not supposed to be here, please move this inside the class

Reply:
Sure